template/actions/color.js

```diff
@@ -0,0 +1,6 @@
+const c = ['#e6ef9f', '#a09fef', '#9fefa1', '#ef9f9f']
+const colors = c.concat(c)
+module.exports = {
+  init: () => colors[0],
+  advance: (color) => colors[colors.indexOf(color) + 1]
```

Comment:
Any possibility of an off by one error?

Reply:
nope, it's supposed to advance by one, so this isn't really magic numbers